changelog.md

```diff
@@ -19,9 +46,56 @@ If the limit of concurrent requests is reached, a HTTP 503 Service Temporarily U
 which means that this operation can be retried.
 You can turn on this feature by setting the parameter `--http_max_concurrent_requests`.
 
+#### Serialize all concurrent change requests
+ 
+In former versions of Marathon it was possible that multiple concurrent changes of the same application definition
+would lead to an inconsistent state of that application. All changes of groups or application definitions now
+get serialized, so the resulting state is always consistent.
+
+#### Enhance the reported metrics 
+
+The metrics endpoint (`/metrics`) gives valuable insight into the system behavior.
+We now have meters for all service objects as well as the number applications and groups in the system.
+
+#### Decouple API (un)marshalling types from internal models
+
+With this change we can control API changes and are free to change internal models.
+This will change nothing in the current API, but will help a lot while we create the v3 API. 
+
+#### Added integration tests for all API endpoints
+
+We now have an integration test suite, that covers all endpoints with almost all possible parameter combinations.
+This test suite runs with every commit on every pull request.
+This test suite is our safety net to ensure compatibility while introducing new functionality in future versions.
+
+#### Code coverage reporting using coveralls
+
+See our github page https://github.com/mesosphere/marathon or go directly to  
+https://coveralls.io/github/mesosphere/marathon to see our code coverage reporting.
+With every Pull Request we try to make sure, this coverage will be increased.  
+
+#### Improved the release process of Marathon itself
+
+Several changes are made to the sbt build file, but also to the infrastructure that is performing the release.
+We now have a more reliable way of pushing distributions.
+
+
```

Comment:
After #1807 gets merged, please add:

#### Custom prefix for automatically created environment variables
It is now possible to specify a custom prefix via the `--env_vars_prefix`
command line flag.  this prefix will be added to the name of task's environment
variables created automatically by Marathon (e.g., `PORT`, `PORTS`)

_Note: This prefix will not be added to variables that are already prefixed,
such as `MESOS_TASK_ID` and `MARATHON_APP_ID`